### ISSUE TITLE
tautulli like page

### ISSUE THREAD

**Issue by binarygeek119** [stashapp/stash#21]
link to [https://tautulli.com/](url)
love for  a data brake down of stash
most viewed tags videos 
most played
time viewing videos
total time cents stash created
stuff of that nature 
just some for stats
i use   tautulli  for plex love it on stash

**Comment by Leopere** [comment]
I've been told that we're averse to centralized tracking so I imagine that we won't be focusing on this in the future. #341 may be related.

**Comment by seducingbits** [comment]
I think OP meant a statisctics page for your own stash instance, not a centralized hub which show statistics for all users.

**Comment by binarygeek119** [comment]
> I think OP meant a statisctics page for your own stash instance, not a centralized hub which show statistics for all users.

yes thats it exactly just be able to view your own stats not for stashdb to see what im playing just the admin to see whats being watch or how populer X is (lesbain,trans,anal,etc)

**Comment by DogmaDragon** [comment]
https://discourse.stashapp.cc/filter?q=tag%3Aplugin%2Bstatistics
